packages/web-react/src/components/Collapse/README.md

```diff
@@ -5,56 +5,104 @@
+| `id`                      | `string`                        | `<random>` | no       | Component id                     |
+| `isCollapsed`             | `boolean`                       | -          | no       | Is collapsed on init             |
+| `collapsibleToBreakpoint` | `'mobile', 'tablet', 'desktop'` | -          | no       | Handle for responsive breakpoint |
+| `UNSAFE_className`        | `string`                        | -          | no       | Wrapper custom classname         |
+| `UNSAFE_style`            | `CSSProperties`                 | -          | no       | Wrapper custom style             |
+
+## Uncontrolled Collapse
+
+### Basic
+
+```javascript
+import React, { useState } from 'react';
+import { Button, UncontrolledCollapse } from '@lmc-eu/spirit-web-react/components';
+
+// ...
+
+<UncontrolledCollapse
   id="CollapseExample"
-  renderToggle={({ trigger }) => (
-    <button className="d-tablet-none" {...trigger}>
-      ...
-    </button>
+  renderToggle={({ collapsed, ...restProps }) => (
+    <Button {...restProps}>Collapse Trigger ({collapsed ? 'Open' : 'Closed'})</Button>
   )}
-  collapsibleToBreakpoint="tablet"
 >
   ...
-</Collapse>
+</UncontrolledCollapse>;
 ```
 
-### Hide on close
-
 ```javascript
-<Collapse id="CollapseExample" renderToggle={({ trigger }) => <button {...trigger}>...</button>} hideOnCollapse>
+import React, { useState } from 'react';
+import { Button, UncontrolledCollapse } from '@lmc-eu/spirit-web-react/components';
+
+// ...
+
+<UncontrolledCollapse
+  id="CollapseExample"
+  renderToggle={({ collapsed, ...restProps }) => (
+    <Button {...restProps}>Collapse Trigger ({collapsed ? 'Open' : 'Closed'})</Button>
+  )}
+  hideOnCollapse
+>
   ...
-</Collapse>
+</UncontrolledCollapse>;
 ```
 
 ## Props
 
-| Prop name                 | Type                                                        | Default    | Required | Description                            |
-| ------------------------- | ----------------------------------------------------------- | ---------- | -------- | -------------------------------------- |
-| `id`                      | `string`                                                    | `<random>` | no       | Component id                           |
-| `isCollapsed`             | `boolean`                                                   | -          | no       | Is collapsed on init                   |
-| `renderTrigger`           | `(render: CollapseRenderProps) => ReactNode`                | -          | no       | Properties for trigger render          |
-| `elementType`             | `'div','article','section','main','header','footer','span'` | -          | no       | Wrapper element type                   |
-| `contentElementType`      | `'div','article','section','main','header','footer','span'` | -          | no       | Wrapper element type                   |
```

Comment:
Not sure If we should allow this. For example there can be a mixture of elements, eg. header in span or main in the header or other combinations. What do you think @adamkudrna?

Reply:
I think we should remain with changing the outer element only.